src/fileActions.jsx

```diff
@@ -184,11 +169,11 @@
         const selectedItem = Array.isArray(selected)
             ? selected[0]
             : selected;
-        if (selectedItem.type === "file") {
+        if (selectedItem.type === "reg") {
             modalTitle = cockpit.format(_("Force delete file $0?"), selectedItem.name);
-        } else if (selectedItem.type === "link") {
+        } else if (selectedItem.type === "lnk") {
             modalTitle = cockpit.format(_("Force delete link $0?"), selectedItem.name);
-        } else if (selectedItem.type === "directory" || selectedItem.items_cnt) {
+        } else if (selectedItem.type === "dir" || selectedItem.items_cnt) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-210-20240130-133717-6c7b7601-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)